src/EnergyPlus/WeatherManager.cc

```diff
@@ -4135,7 +4135,7 @@ namespace WeatherManager {
         Real64 const ZHGlobalSolarConstant(1355.0);
         static ObjexxFCL::gio::Fmt EnvDDHdFormat(
             "('! <Environment:Design Day Data>, Max Dry-Bulb Temp {C}, ',   'Temp Range {dC}, Temp Range Ind Type, ',   "
-            "'Hum Ind Value at Max Temp, Hum Ind Type,Pressure {Pa}, ',   'Wind Direction {deg CW from N}, ',    'Wind "
+            "'Hum Ind Type, Hum Ind Value at Max Temp, Pressure {Pa}, ',   'Wind Direction {deg CW from N}, ',    'Wind "
```

Comment:
EIO header change, to match how it's layed out later (but not reported until now)

Reply:
Good to add this. Since the units are variable and can't be declared in the header, there should be an additional column for "Hum Ind Value Units" (or similar).  This will ensure that numbers and strings are kept separate for easier parsing of the eio.